src/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+    "fmt";
+    "os";
+    "time";
+    "strconv";
+    "encoding/json";
```

Comment:
A gente precisa desses `;` no final das linhas? O padrão no Go tb é ordenado alfabeticamente, não?

```suggestion
    "encoding/json"
    "fmt"
    "strconv"
    "time"
    "os"
```